pipeline/helpers/data_loaders.py

```diff
@@ -31,8 +33,14 @@ def load(path, n_samples=None, ids=None, n_ids=None):
 
 
 def load_EIA():
+    destination = DATA_DIR + "/eia_electricity_hourly.csv"
+    if not os.path.exists(destination):
+        dataset = load_dataset("HPDNorm/eia_electricity_hourly")
```

Comment:
I think you can specify `split="all"` to load a full dataset so you don't have to reload it from a file again in the return